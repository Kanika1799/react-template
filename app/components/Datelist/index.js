/**
 *
 * Datelist
 *
 */

import React, { memo } from 'react';
// import PropTypes from 'prop-types';
// import styled from 'styled-components';
// import { FormattedMessage as T } from 'react-intl';

function Datelist() {
  return <div data-testid="datelist">{/* <T id="date" values={{ date: dashData.date }} /> */}</div>;
}

Datelist.propTypes = {
  // dashData: PropTypes.object
};

export default memo(Datelist);
