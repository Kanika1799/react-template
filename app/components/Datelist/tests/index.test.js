/**
 *
 * Tests for Datelist
 *
 */

import React from 'react';
// import { fireEvent } from '@testing-library/dom'
import { renderWithIntl } from '@utils/testUtils';
import Datelist from '../index';

describe('<Datelist />', () => {
  it('should render and match the snapshot', () => {
    const { baseElement } = renderWithIntl(<Datelist />);
    expect(baseElement).toMatchSnapshot();
  });

  it('should contain 1 Datelist component', () => {
    const { getAllByTestId } = renderWithIntl(<Datelist />);
    expect(getAllByTestId('datelist').length).toBe(1);
  });
});
