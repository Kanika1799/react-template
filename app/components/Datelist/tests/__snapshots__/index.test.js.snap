// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Datelist /> should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      data-testid="datelist"
    />
  </div>
</body>
`;
